### PR TITLE
Add 3rd AGC data

### PR DESCRIPTION
**Pull request by tthieuhcm**
- [x] Add 1st experiment data
- [x] Clean 1st experiment data
- [x] Add 2nd experiment data
- [x] Clean 2nd experiment data

### CODE REVIEW

File: GH_Lettuce/3rd_AutonomousGreenhouseChallenge_1stExperiment/Koala_images.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+https://drive.google.com/drive/folders/1Os93a9AGK3KB4aKTiXK766us57JyzKsm?usp=share_link
```
Comment (zer0n) [inline GH_Lettuce/3rd_AutonomousGreenhouseChallenge_1stExperiment/Koala_images.txt:1]: `.md` is preferred to `.txt`

File: GH_Lettuce/3rd_AutonomousGreenhouseChallenge_2ndExperiment/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# **International Autonomous Greenhouse Challenge - 3rd edition - 2nd experiment**
```
Comment (zer0n) [inline GH_Lettuce/3rd_AutonomousGreenhouseChallenge_2ndExperiment/README.md:1]: Add info about the images data to README.

Remove full data and images.txt files.

File: GH_Lettuce/3rd_AutonomousGreenhouseChallenge_1stExperiment/README.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
+# **International Autonomous Greenhouse Challenge - 3rd edition - 1st experiment**
```
Comment (zer0n) [inline GH_Lettuce/3rd_AutonomousGreenhouseChallenge_1stExperiment/README.md:1]: Can't use WUR's language. Make it Koidra authentic. It doesn't need to be verbose.

For folder names, use this pattern: `Koidra.AGC2022.1` and `Koidra.AGC2022.2`

Comment (zer0n) [reply GH_Lettuce/3rd_AutonomousGreenhouseChallenge_1stExperiment/README.md:1]: Also add a few public press releases that highlight Koidra's success.
- https://www.wur.nl/en/newsarticle/familiar-face-leads-team-koala-to-win-the-autonomous-greenhouse-challenge-for-a-second-time.htm
- https://www.geekwire.com/2022/how-a-former-microsoft-researcher-used-ai-to-grow-award-winning-lettuce-from-5000-miles-away/

**Comment by tthieuhcm** [comment]
A @zer0n: I addressed all of above comments.

**Pull request merged**